backend/station_manager/app/src/main/java/com/StationManager/app/services/command_listener/ForwardPubSub.java

```diff
@@ -0,0 +1,23 @@
+package com.StationManager.app.services.command_listener;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.JedisPubSub;
+
+import static com.StationManager.app.Api.wsSessions;
+
+public class ForwardPubSub extends JedisPubSub {
+    public static final Logger logger = LoggerFactory.getLogger(ForwardPubSub.class);
+
+    private static void broadcastMessage(String message) {
+        wsSessions.forEach(session -> {
```

Comment:
better not reference objects somewhere from other part of the program, use dependency injection.
So when initiating a pub/sub you would pass `wsSessions` to the constructor.